apps/bttester/syscfg.yml

```diff
@@ -110,11 +118,12 @@ syscfg.vals:
     MSYS_1_BLOCK_COUNT: 100
 
     BLE_MONITOR_RTT: 1
-    CONSOLE_RTT: 0
+    BLE_MONITOR_RTT_BUFFER_SIZE: 4096
```

Comment:
do we want this configuration to be delivered inside BASS test?

Reply:
Removed